opm/polymer/PolymerProperties.hpp

```diff
@@ -147,14 +147,14 @@ namespace Opm
             const auto& plyviscTable = tables->getPlyviscTables().getTable<PlyviscTable>(0);
 
 
-            c_vals_visc_ = plyviscTable.getPolymerConcentrationColumn();
-            visc_mult_vals_ =  plyviscTable.getViscosityMultiplierColumn();
+            c_vals_visc_ = plyviscTable.getPolymerConcentrationColumn().vectorCopy( );
```

Comment:
No need, just curious. Fixate on one if you eventually decide one of them is better.